### PR TITLE
remove password expired logic

### PR DESCRIPTION
**Pull request by artembiserov**
https://github.com/alphagov/signon/issues/524

### CODE REVIEW

**Comment by cbaines** [comment]
Thanks for this @artembiserov :) . Could you add some more information to you commit, particularly about the motivation for making this change (this doesn't have to be complicated)?

File: app/models/event_log.rb [hunk 1]
```diff
@@ -9,7 +9,6 @@ class EventLog < ActiveRecord::Base
     ACCOUNT_UNSUSPENDED                       = LogEntry.new(id: 3, description: "Account unsuspended", require_initiator: true),
     ACCOUNT_AUTOSUSPENDED                     = LogEntry.new(id: 4, description: "Account auto-suspended"),
     MANUAL_ACCOUNT_UNLOCK                     = LogEntry.new(id: 5, description: "Manual account unlock", require_initiator: true),
-    PASSPHRASE_EXPIRED                        = LogEntry.new(id: 6, description: "Passphrase expired"),
```
Comment (benlovell) [inline app/models/event_log.rb]: This ought to stay around otherwise these historical audit entries will be orphaned.

**Comment by matthewford** [comment]
@cbaines the primary motivation was a pair programming test, where I thought I might as well scratch an itch at the same time.

**Comment by matthewford** [comment]
It brings the application inline with the new NCC guidance around requiring password rotation.

**Pull request closed without merging**